src/locales/fr.json

```diff
@@ -430,7 +430,8 @@
     "numberOfApplications" : "%{smart_count} application |||| %{smart_count} applications",
     "applications": "Applications",
     "data": "Données",
-    "open": "Ouvrir"
+    "open": "Ouvrir",
+    "access": "Les applications disposant de cette permission peuvent accéder à votre/vos "
```

Comment:
we need to use plural form using smart count (like line 430)

Reply:
I think smart_count counts the number of applications. This is not the same usage here. What I can try to do is checking if the last letter of the Permission is an "s". If this is an "s" then I write "vos" else I write "votre".

Reply:
if our PM agrees to use votre/vos, let's keep it